code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -64,7 +65,7 @@ Difficulty: Hard
 	/// Does it have a portal to the funny second life arena created?
 	var/obj/effect/portal/redspace/second_life_portal
 	/// Max healing bubblegum can get from being enraged
-	var/maximum_enraged_healing = 500
+	var/maximum_enraged_healing = MAXIMUM_ENRAGED_HEALLING
```

Comment:
I meant getting rid of this variable totally, since I can't see anywhere its ever edited

Reply:
good point, sorry for being dumb

Reply:
Done!